### PR TITLE
Better support for regional formatting guidelines

### LINKED ISSUE #111: Better support for regional formatting guidelines
Depending on national conventions, currency symbol can be used before, between or after the numeric amounts. For currencies in English-speaking countries and in most of Latin America, the symbol is placed before the amount, as in $20.50. In most other countries, including many in Europe, the symbol is placed after the amount, as in 20,50€.

To better support developers from all around the world, especially from underrepresented communities, string interpolation should account for those variations:

```js
const const name = "world"!
print(`Hello ${name}!`)!
print(`Hello {name}€!`)!


// Whitespace is allowed (when autoformatting, U+2009 THIN SPACE should be used):
print(`Hello {name} €!`)!
```

The symbol for the Cape Verdean escudo is [placed in the decimal separator position](https://en.wikipedia.org/wiki/Cape_Verdean_escudo), as in 2$50. Developers from the Republic of Cape Verde could benefit from this syntax:

```js
class Player {
   const var health = 10!
}
const var player = new Player()!

print(`Your health is: {player$health}!`)!
```

### PR DESCRIPTION
**Pull request by notpushkin**
Closes #111

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -242,7 +242,16 @@ Please remember to use your regional currency when interpolating strings.
 const const name = "world"!
 print("Hello ${name}!")!
 print("Hello £{name}!")!
-print("Hello €{name}!")!
+
+// Follow the local typographical norms:
+print("Hello {name} €!")!
```
Comment (TodePond) [inline README.md:247]:
```suggestion
\```

Make sure you follow your local typographical norms.

```java
print("Hello {name}€!")!
```

^ I think this is pretty fun, so we could give it some more breathing room. (Note-to-self: Might need to remove my backslash)

**Review by TodePond — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (notpushkin) [reply README.md:247]: Sounds good!

Backslash seems to mangle it, yeah – if you get rid of it I can commit this (or I can just copy it over myself)

Comment (TodePond) [reply README.md:247]: I wonder what happens if you click 'Commit suggestion' now. I don't think github PRs were designed for this :)

Comment (notpushkin) [reply README.md]: pow!

**Comment by TodePond** [comment]
Oh dear! :)
we are pioneering & learning at least

**Comment by notpushkin** [comment]
Peak social coding experience :D

Wanna do another one?

**Comment by TodePond** [comment]
Nice one!

**Pull request merged**